.travis.yml

```diff
@@ -30,7 +30,9 @@ script:
 deploy:
   - provider: script
     skip_cleanup: true
-    script: npm publish
+    script:
+      - test v"$(jq -r .version package.json)" = "$TRAVIS_TAG"
```

Comment:
We may want to add a test for something like `if test "$TRAVIS_NODE_VERSION" = 10` also, or it might try and deploy with each Node.js version that it runs, and one of them will fail.

Reply:
Sorry, that was pseudocode. I think this will fail unless you remove the `if` and just have `test` as the first word in the script. Otherwise the shell will want to match then `if` to a `then / fi` combination

Reply:
Oh, also note that we can't compare string to a number or it might break. Not sure how `"10" = 10` will evaluate.